src/installer.ts

```diff
@@ -53,18 +190,23 @@ export async function getNode(
     versionSpec = resolveLtsAliasFromManifest(versionSpec, stable, manifest);
   }
 
-  if (isLatestSyntax(versionSpec)) {
+  // TODO: 183-189 and 193-194   seems to be the same. Why do we need them?
```

Comment:
On lines183-189 we download manifest file and try to resolve the version by lts syntax. On lines 193-194 the action does not download manifest file. The action calls dist directly to get versions from index.json. Keep in mind that dist is more likely to hit rate limit.